common/service/dynamicconfig/constants.go

```diff
@@ -70,6 +70,7 @@ var keys = map[Key]string{
 	EnableParentClosePolicyWorker:       "system.enableParentClosePolicyWorker",
 	EnableStickyQuery:                   "system.enableStickyQuery",
 	EnablePriorityTaskProcessor:         "system.enablePriorityTaskProcessor",
+	EnableAuthorization:                 "system.enableAuthorization",
```

Comment:
Missing something? You need to use it somewhere, right?

Reply:
This will be used in authorizer implementation. In this case, it's uber internal server.